Makefile

```diff
@@ -12,7 +12,7 @@ help: ## Show this help
 	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "  \033[32m%-18s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
 
 lint: ## Runs the linting tool
-	docker run --rm -u ${user} -v "$(shell pwd):/docs:ro" -e INPUT_IGNORE=${ignored} avtodev/markdown-lint:v1.5 \
+	docker run --rm -u ${user} -v "$(shell pwd):/docs" -w /docs -e INPUT_IGNORE=${ignored} ghcr.io/shyim/spellcheck-github-actions:add-arm64 \
```

Comment:
Should we also use the same image for `make fix`, and make the image configurable with env var, defaulting to the official image?

Reply:
done :)